commodore/catalog.py

```diff
@@ -124,3 +129,42 @@ def catalog_list(cfg):
             click.echo(f" - {display_name}")
         else:
             click.echo(catalog_id)
+
+
+def list_components(config: Config, cluster_id: str, output: Optional[str] = None):
```

Comment:
This method doesn't really belong in `catalog.py`, since it doesn't interact with the cluster catalog at all. If I had to place it somewhere, I'd probably put it in `cluster.py`.